airflow/www/static/js/connection_form.js

```diff
@@ -59,6 +59,8 @@ function getControlsContainer() {
 $(document).ready(function () {
   const fieldBehavioursElem = document.getElementById('field_behaviours');
   const config = JSON.parse(decode(fieldBehavioursElem.textContent));
+  const form = document.getElementById('model_form');
```

Comment:
Maybe add a comment to make the intensions here clear?
```suggestion
  // Prevent login/password fields from triggering browser auth extensions
  const form = document.getElementById('model_form');
```

Reply:
Good idea. Added.